docs/index.fsx

```diff
@@ -47,66 +66,69 @@ DiffSharp provides world-leading automatic differentiation capabilities for tens
 
-You can execute this page as an interactive notebook running in your browser, or download it as a script or .NET Interactive Jupyter notebook, using the buttons [![Binder](img/badge-binder.svg)](https://mybinder.org/v2/gh/diffsharp/diffsharp.github.io/master?filepath={{fsdocs-source-basename}}.ipynb) 
-[![Script](img/badge-script.svg)]({{fsdocs-source-basename}}.fsx) 
-[![Script](img/badge-notebook.svg)]({{fsdocs-source-basename}}.ipynb) on the top of the page. This applies to all documentation pages.
+## Interactive Notebooks
 
-If using Visual Studio Code you can download, edit and execute these notebooks using [the .NET Interactive Notebooks for VS Code](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.dotnet-interactive-vscode).
+All documentation pages in this website are interactive notebooks which you can execute directly in your browser without installing anything in your local machine.
 
-First reference the package:
+Using the buttons [![Binder](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/DiffSharp/diffsharp.github.io/blob/master/{{fsdocs-source-basename}}.ipynb) [![Binder](img/badge-binder.svg)](https://mybinder.org/v2/gh/diffsharp/diffsharp.github.io/master?filepath={{fsdocs-source-basename}}.ipynb) on the top of each page, you can execute the page as an interactive notebook running on cloud servers provided by [Google Colab](https://colab.research.google.com/) and [Binder](https://mybinder.org/).
 
-    #r "nuget: DiffSharp-lite, {{fsdocs-package-version}}"
+Using the buttons [![Script](img/badge-script.svg)]({{fsdocs-source-basename}}.fsx) 
+[![Script](img/badge-notebook.svg)]({{fsdocs-source-basename}}.ipynb) you can also download a page as a script or an interactive notebook, which you can execute locally in [Jupyter](https://jupyter.org/) or [Visual Studio Code](https://code.visualstudio.com/) using [dotnet interactive](https://github.com/dotnet/interactive).
 
-or for LibTorch support:
+## Example
 
-    #r "nuget: DiffSharp-cpu, {{fsdocs-package-version}}"
+Define and add two tensors:
 *)
 
 open DiffSharp
 
-(** 
-Configure:
-*)
-dsharp.config(dtype=Dtype.Float32, device=Device.CPU, backend=Backend.Reference)
-
-(** 
-Defining and adding two tensors:
-*)
-let t1 = dsharp.tensor [ 0.0 .. 0.2 .. 1.0 ]
-let t2 = dsharp.tensor [ 0, 1, 2, 4, 7, 2 ]
+let t1 = dsharp.tensor [ 0.0 ..0.2.. 1.0 ] // Gives [0., 0.2, 0.4, 0.6, 0.8, 1.]
+let t2 = dsharp.tensor [ 1, 2, 3, 4, 5, 6 ]
 
 t1 + t2
+(*** include-it ***)
 
 (** 
-Computing a convolution:
+Compute a convolution:
 *)
 let t3 = dsharp.tensor [[[[0.0 .. 10.0]]]]
-let t4 = dsharp.tensor [[[[0.0 ..0.1 .. 1.0]]]]
+let t4 = dsharp.tensor [[[[0.0 ..0.1.. 1.0]]]]
 
 t3.conv2d(t4)
+(*** include-it ***)
 
 (** 
 Take the gradient of a vector-to-scalar function:
 *)
 
-let f (x: Tensor) = x.exp().sum()
+let f (x:Tensor) = x.exp().sum()
```

Comment:
Recommend space after `:` as standard formatting

Reply:
Personally I find `x:Tensor` to be much smoother (doesn't break the flow), but if we adopt `x: Tensor` as a style, I can adapt to that. It's a minor thing.